CHANGELOG.md

```diff
@@ -30,6 +30,7 @@ The types of changes are:
 
 ### Changed
 - Update when GPP API reports signal status: ready [#4635](https://github.com/ethyca/fides/pull/4635)
+- Modify `fides user login` to not store plaintext password in `~/.fides-credentials` [#4661](https://github.com/ethyca/fides/pull/4661)
```

Comment:
Sorry Tariq, I should have been clearer, this changelog entry should go up further in the `[Unreleased]` section, not here in the `[2.31.0]` section. 

Because of the timing, this particular change will be included in a later release than `2.31.0`.